lib/bindings/Cargo.toml

```diff
@@ -11,22 +11,29 @@ path = "uniffi-bindgen.rs"
 name = "breez_sdk_liquid_bindings"
 crate-type = ["staticlib", "cdylib", "lib"]
 
+[features]
+default = ["uniffi-28"]
+uniffi-25 = ["uniffi_25", "uniffi_bindgen_25", "uniffi_bindgen_kotlin_multiplatform"]
+uniffi-28 = ["uniffi_28", "uniffi_bindgen_28"]
+
 [lints]
 workspace = true
 
 [dependencies]
 anyhow = { workspace = true }
 breez-sdk-liquid = { path = "../core" }
 log = { workspace = true }
-uniffi = { workspace = true, features = [ "bindgen-tests", "cli" ] }
-# Bindgen used by KMP, version has to match the one supported by KMP
-uniffi_bindgen = "0.25.2"
-uniffi_bindgen_kotlin_multiplatform = { git = "https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings", rev = "e8e3a88df5b657787c1198425c16008232b26548" }
+uniffi_25 = { package = "uniffi", version = "0.25.2", features = [ "bindgen-tests", "cli" ], optional = true }
+uniffi_28 = { package = "uniffi", version = "0.28.0", features = [ "bindgen-tests", "cli" ], optional = true }
+uniffi_bindgen_25 = { package = "uniffi_bindgen", version = "0.25.0", optional = true }
```

Comment:
Should this match the `uniffi_25` version (`0.25.2`)?

Reply:
Ideally, yeah. Updated [8136411](https://github.com/breez/breez-sdk-liquid/pull/766/commits/81364115edfc684ba034d11b5e21178097a00445)